mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -35,5 +36,8 @@ def generate_refund_file(request):
     client = api_client.get_connection(request)
     client.bank_admin.transactions.patch(refunded_transactions)
 
+    # create file record
+    post_new_file(request, 'ACCESSPAY', [t['id'] for t in refunded_transactions])
```

Comment:
(To be clear: I'm suggesting that we use that generic term in the DB/API, and that the bank admin app is responsible for knowing that, at this moment in time, the format a “refund journal” should be downloadable as is an ADI. Does that make sense?)